acpi/Cargo.toml

```diff
@@ -10,6 +10,9 @@ license = "MIT/Apache-2.0"
 edition = "2018"
 
 [dependencies]
-log = "0.4"
 bit_field = "0.10"
 rsdp = { version = "2", path = "../rsdp" }
+
+[features]
+default = []
```

Comment:
How do you feel about including `alloc` as a default feature - it feels like this would be the standard use of the library (e.g. within a kernel with some sort of allocator), and the no-alloc usage would be a bit more niche?

Reply:
Yes, I agree with this. How about `"allocator_api"` and `"auto_global_alloc"`? I'm not sure for the latter feature name.

Reply:
+1 for `allocator_api`, maybe just `global_alloc` for the latter? Nit either way though